CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## Unreleased
+
+- Add parameters kibana_user, kibana_group
+
```

Comment:
Please don't update the changelog: it is updated when we release a new version.

Reply:
ok